clodius/tiles/bigbed.py

```diff
@@ -0,0 +1,407 @@
+import bbi
+import functools as ft
+import logging
+import numpy as np
+import pandas as pd
+import re
+import sys
+import random
+
+from concurrent.futures import ThreadPoolExecutor
+
+MAX_THREADS = 4
+TILE_SIZE = 1024
+MIN_ELEMENTS = 1
+MAX_ELEMENTS = 50
+
+logger = logging.getLogger(__name__)
+
+range_modes = {}
+range_modes['significant'] = {'name': 'Significant', 'value': 'significant'}
+
+
+def get_quadtree_depth(chromsizes):
```

Comment:
This function and a few of the others look like they're identical to the ones in `clodius.tiles.bigWig`. Maybe use those directly with an import or factor out to a common file?

Reply:
I'll import from `clodius.tiles.bigWig`.